content/post/88-js-ipfs-0.36.md

```diff
@@ -0,0 +1,175 @@
+---
+date: 2019-05-21
+url: 88-js-ipfs-0-36
+title: js-ipfs 0.36.0 released
+author: Alan Shaw
+---
+
+> URL safe CIDs, refs commands, DOM File support and more!
+
+# 🔦 Highlights
+
+## 🧬 Base32 encoding for v1 CIDs
```

Comment:
When will the default of a ipfs add be v1 CIDs?

Reply:
Could be as soon as next quarter - it'll be a lot easier now

Reply:
Can it be before IPFS Camp?

Reply:
❤️ I would like that...

It makes sense to get base32-able Peer IDs in before though. We can't really shout about being able to use CIDs in domains until we have that.

We should also orchestrate a release with go-ipfs for interop. I don't know how feasible it is to get this in from the Go side. I know this is a big breaking change for them (ahem, 0.5.0?). Personally, I'm feeling pretty stacked atm and I know it isn't the current plan.

cc @Stebalien @lidel

Reply:
We were discussing base32-able Peer IDs on Go Weekly, and we are on route to ship support for `https://{libp2p-key-in-cidv1b32}.ipns.dweb.link` as opt-in before IPFS Camp.
Progress is tracked in https://github.com/ipfs/go-ipfs/issues/5287